src/main/java/com/google/devtools/build/lib/bazel/bzlmod/IndexRegistry.java

```diff
@@ -368,6 +376,21 @@ private RepoSpec createLocalPathRepoSpec(
         .build();
   }
 
+  private List<String> getMirrorUrls(Optional<BazelRegistryJson> registry) {
+    return registry.map(r -> r.mirrors)
```

Comment:
IMO revert this part of the change -- we no longer check for `mirrors` being null, we lost a code comment, and this now throws an unchecked exception on MalformedURLException causing a Bazel crash.

Reply:
Done.